recipes/default.rb

```diff
@@ -119,4 +119,5 @@
       eof
     }
   EOF
+  not_if { File.exist?("#{setup_root}/#{node['android-sdk']['name']}-#{node['android-sdk']['version']}") }
```

Comment:
what if we want to add a component to the list? will this not if block stops updating the new component